salt/modules/libcloud_loadbalancer.py

```diff
@@ -380,3 +452,17 @@ def _simple_member(member):
         'balancer': _simple_balancer(member.balancer),
         'extra': member.extra
     }
+
+
+def _sanitize_kwargs(kwargs):
```

Comment:
Yeah, I think it's a relatively recent function. Plus it's buried in the massive `salt.utils.__init__.py` file. :)